### DISCUSSION TITLE
TAB key not working in Windows

### DISCUSSION THREAD

**Discussion by jeffintheusa** [daniele77/cli#182] [General]
Hi,

I'm running the examples/simplelocalsession in Visual Studio 2022 and the TAB button does not complete the command, instead it moves the cursor the same as if a normal TAB operation was performed. Any ideas on how to correct this?

Thank you,

Jeff.

**Comment by daniele77** [comment]
Please, see this answer (the first part): https://github.com/daniele77/cli/discussions/148#discussioncomment-2744818
